src/types/User.ts

```diff
@@ -68,7 +68,7 @@ export interface UserStatus {
 }
 
 export interface UserLocation {
-  lastSeenIn: { [key: string]: number };
+  lastSeenIn: string | null;
```

Comment:
Same point on backward compatibility as in `LocationUpdateData `

Reply:
Changed in https://github.com/sparkletown/sparkle/pull/2202/commits/fa9ecad2e4a8b13737c78d565396fd369a53a2c1